### DISCUSSION TITLE
[request] Add speed limit for file transfering

### DISCUSSION THREAD

**Discussion by kiohime** [doublecmd/doublecmd#1090] [Suggestions]
### Description

1.10 doublecmd, windows 10

Please add ability to set speed limit for file transfering operations like downloading, copying, etc. 
In Total Commander when you copy file, you have specific field, where you can set limit for file download.
It is very useful feature, when you have many PC in one LAN and you dont want to take all bandwidth of network.

**Comment by ags1234** [comment]
+1
I would also use this feature if it were available.

**Comment by oasisindesert** [comment]
+1 Waiting for this useful feature to prevent system hanging on copying large files.

**Comment by ags1234** [comment]
And to avoid the situation where DC's network transfer occupies all the bandwidth, disrupting other traffic.

**Comment by duvte** [comment]
I have needed this option for a long time to set the speed limit under Linux

**Comment by ric-cz** [comment]
Hello, is there any update regarding this request? I also vote for this feature. I often transfer huge amout of data over network and need to reduce the use bandwidth.

**Reply by alexx2000** [reply]
Hi, no.

**Reply by 1337gut** [reply]
Still no update? Just asking, don't stress.

**Reply by alexx2000** [reply]
No
